packages/test-utils/src/configure.ts

```diff
@@ -0,0 +1,13 @@
+export interface Configuration {
+  emotion: boolean;
```

Comment:
Should we document the usage of the prop?

```suggestion
  /**
   * The emotion wrapper is optional.
   * If you package uses emotion, install ... and set this to true.
   */
  emotion: boolean;
```